vstools/enums/color.py

```diff
@@ -988,6 +988,13 @@ def _missing_(cls: type[ColorRange], value: Any) -> ColorRange | None:
     def from_res(cls, frame: vs.VideoNode | vs.VideoFrame) -> ColorRange:
         """Guess the color range from the frame resolution."""
 
+        from ..utils import get_var_infos
+
+        fmt, w, h = get_var_infos(frame)
```

Comment:
can the fixes be moved to another PR? like a generic bugfix/typos one

Reply:
Some of the tests fail without the changes. I suppose I can temporarily @ignore those and unignore in the fix PR.